napari_aicssegmentation/view/batch_processing_view.py

```diff
@@ -29,17 +34,71 @@ def _setup_ui(self):
                 mode=FileInputMode.FILE, filter="Json file (*.json)", initial_text="Load a JSON workflow file..."
             ),
         )
+        row1.widget.file_selected.connect(self.workflow_selected)
         channel_input = QLineEdit("0")
         channel_input.setValidator(QIntValidator(bottom=0))
         row2 = FormRow("2.  Structure channel index:", channel_input)
         row3 = FormRow(
             "3.  Input directory:", FileInput(mode=FileInputMode.DIRECTORY, initial_text="Select a directory...")
         )
+        row3.widget.file_selected.connect(self.input_folder_selected)
         row4 = FormRow(
             "4.  Output directory:", FileInput(mode=FileInputMode.DIRECTORY, initial_text="Select a directory...")
         )
+        row4.widget.file_selected.connect(self.output_folder_selected)
 
         # TODO figure out how to merge cells to display help line (use GridLayout instead of FormLayout?)
```

Comment:
The wireframes have a help line displayed under row 3 that says _Supported file formats: .tiff, .ome.tiff [...]_. Doesn't look like this is included anywhere? Did you all decide to leave it out?

Reply:
I put this in last minute at the meeting, the new code I am pushing up will include this. We also decided to move this down to the very bottom (right above the buttons) in our meeting last Friday